### ISSUE TITLE
Dim the non-active split?

### ISSUE THREAD

**Issue by lougreenwood** [nvim-focus/focus.nvim#137]
Just an idea, but I use [`tint.nvim`](https://github.com/levouh/tint.nvim) to dim splits which are not the currently active ones. Given the feature set and (what I infer) is the goal of creating a tool to help a user focus on the current file, did you consider adding integrating something like this into `focus`?

**Comment by willothy** [comment]
Not sure what the maintainers/creator will think, but imo it's better to keep features isolated in their own plugins. Adding `tint`'s featureset into `focus` would basically mean building a whole separate plugin into this codebase (`tint` contains ~700LOC), which would add considerable maintenance burden. Why not just use both?

**Comment by lougreenwood** [comment]
Yep, agreed WRT to using both, just wanted to "highlight" a use case which could also fall into the scope of what `focus` kinda does  - especially with the `win highlight` options (although I don't see what that actually does! 😀).

Feel free to close as not planned if you want 👍.

**Comment by willothy** [comment]
Totally! Would definitely be cool to have, and you're right that it does fall somewhat in the scope of what focus tries to do. I'm not a maintainer so I can't close issues, but anyways it'd be good to get other opinions on this :)
